### ISSUE TITLE
PTN004341755 (PTHR12192)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-annotation#4528]
* **PTHR ID & PTN node:**
PTN004341755 (PTHR12192)

* **Sequences with problematic annotation (ID + gene/protein name):**
CATION TRANSPORT PROTEIN CHAC-RELATED (PTHR12192) comprises human CHAC1 and CHAC2, which are Glutathione-specific gamma-glutamylcyclotransferases (https://www.uniprot.org/uniprotkb/Q9BUX1/entry & https://www.uniprot.org/uniprotkb/Q8WUX2/entry) and are annotated at UniProt with EC:4.3.2.7 (= glutathione-specific gamma-glutamylcyclotransferase)

But this PAINT node adds the related term "gamma-glutamylcyclotransferase activity (GO:0003839) (= EC:4.3.2.9) rather than "glutathione specific gamma-glutamylcyclotransferase activity (GO:0061928) (= EC:4.3.2.7)"

* **Type of Issue: Erroneous source or erroneous propagation, or other issue**
Erroneous source?

**Comment by marcfeuermann** [comment]
What can I say, we have 5 primary GO annotations to  gamma-glutamylcyclotransferase activity (GO:0003839) from E.coli, yeast and plants, and 0 annotations to glutathione-specific gamma-glutamylcyclotransferase (GO:0061928).
Probably the primary GO annotation for the human proteins has not been done yet. 
In PAN-GO, we cannot PAINT a term we do not have, that's why we used the more general term. I will ask UniProt to make the GO annotations. Once they will be available for PAINTing, I will be able to make the change.
Thanks a lot,
Marc.

**Comment by sjm41** [comment]
Sorry, I wasn't blaming the PAINT annotators here!! Just pointing out that the PAINT propagation is at odds with the UniProt annotation of the human proteins.  As I say, I think the root problem here is an erroneous (or non-specific) source annotation. I've just added IDA annotations for GO:0061928 to the two human proteins from PMID: 27913623 in P2GO.

**Comment by marcfeuermann** [comment]
There is no reason to be sorry, I was the one who was a bit cynical in my message and it is my turn to apologize. We have been discussing the possibility of annotating directly with PAINT for some time and your request made me think of this. It could save us a lot of time and frustration ;-) 
I had thought of doing the IDA annotations. Thanks for taking care of it. Your input are always welcome. Have a nice evening.

**Comment by sjm41** [comment]
No problem Marc! This might be a case where the E.coli and maybe the yeast GO annotations to GO:0003839 are correct, whereas metazoa should have GO:0061928. I'm afraid I haven't checked.
Note the two GO terms are sister terms rather than parent-child in the GO, so you may need to annotate to the shared parent if you keep a single annotation node.

```
|__amidine-lyase activity
           |__gamma-glutamylcyclotransferase activity
           |__glutathione specific gamma-glutamylcyclotransferase activity
```

**Comment by huaiyumi** [comment]
I checked the Interpro for the entry Glutathione-specific gamma-glutamylcyclotransferase (https://www.ebi.ac.uk/interpro/entry/InterPro/IPR006840/). It cross-references to GO term gamma-glutamylcyclotransferase activity. In the cross reference section, it cross references to both ECs (4.3.2.7 and 4.3.2.9). 
Also the term glutathione specific gamma-glutamylcyclotransferase activity (GO:0061928) was created in 2017 in GO, but so far there is not a single experimental annotation to the term. There are some IEA to it though.

**Comment by sjm41** [comment]
Related ticket for InterPro (already fixed, but not yet shown on website) is https://github.com/geneontology/go-annotation/issues/4527.

IPR006840 is called "Glutathione-specific gamma-glutamylcyclotransferase" and the entry description says it applies to the "ChaC family of proteins".

All 17 Swiss-Prot proteins with IPR006840 are ChaC proteins, and all but 3 are manually annotated with EC:4.3.2.7 (=GO:0061928 glutathione specific gamma-glutamylcyclotransferase activity):
https://www.uniprot.org/uniprotkb?facets=reviewed%3Atrue&query=%28xref%3Ainterpro-IPR006840%29

The only exceptions are these three Arabidopsis proteins, currently annotated to EC:4.3.2.9 (= GO:0003839 gamma-glutamylcyclotransferase activity):
Gamma-glutamylcyclotransferase 2-1 [Q8GY54](https://www.uniprot.org/uniprotkb/Q8GY54/entry) from PMID: 24214398
Gamma-glutamylcyclotransferase 2-2 [Q84MC1](https://www.uniprot.org/uniprotkb/Q84MC1/entry) from PMID: 25716890
Gamma-glutamylcyclotransferase 2-3 [Q84QC1](https://www.uniprot.org/uniprotkb/Q84QC1/entry) from PMID: 25716890

I think these need correcting to use the other EC/GO term - I'll contact UniProt about that.
E.g. abstract of PMID: 25716890 says: "We show through in vivo functional assays in yeast that all three members of the ChaC/GCG subfamily show significant activity towards glutathione but no detectable activity towards γ-glutamyl methionine. Biochemical characterization of the purified recombinant enzymes GGCT2;2 and GGCT2;3 further confirmed that they act specifically to degrade glutathione to yield 5-oxoproline and Cys-Gly peptide and show no significant activity towards γ-glutamyl cysteine."

Also note I recently added some manual annotations to the human proteins for the "glutathione specific gamma-glutamylcyclotransferase activity (GO:0061928)" term.

As for the PAINT node, as I said above, it may be more appropriate to annotate to the parent term for the whole family, or else annotate using the specific terms to subfamilies.

**Comment by sjm41** [comment]
Hi @marcfeuermann just checking - Did you fix the PAINT annotation here, or are you waiting on the IDA annotations becoming available (or other feedback)?

**Comment by marcfeuermann** [comment]
Hi Steven,
It is not done yet. I check regularly if the annotations are available for PAINTing. It takes up to 6 months sometimes. This is an issue we are trying to work on. There are 2 issues on hold like that. I do not have access to PAINT today, but I will check again tomorrow.
I will close the ticket as soon as it will be possible to solve it.
Best regards.
Marc.

**Comment by marcfeuermann** [comment]
Still not available for PAINTin :-(

**Comment by marcfeuermann** [comment]
Finally, I could make the changes and close this ticket. Thank you for your patience ;-)

**Comment by sjm41** [comment]
Excellent, thanks!
